### DISCUSSION TITLE
Can't create more than 8 Web Panel Tabs

### DISCUSSION THREAD

**Discussion by innocentliny** [zen-browser/desktop#2739] [General]
Hi,

I know there was an issue about this topic [here](https://github.com/zen-browser/desktop/issues/1397).
But I don't know why and can't view linear zen-browser workspace.
So I'd like to know whether 8 limit is the design purpose or not, and should it be enlarge or unlimited.

Thank you.

**Comment by Eragonfr** [comment]
There is a setting in the advanced options that can be changed in about:config.
`zen.sidebar.max-webpanels` defaults to `8`.
